docs/pages/enroll-resources/machine-id/deployment/bitbucket.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Deploying Machine ID on Bitbucket Pipelines
+description: How to install and configure Machine ID on Bitbucket Pipelines
+---
+
+In this guide, you will configure Machine ID's agent, `tbot`, to run within a
+Bitbucket Pipelines workflow. The bot will be configured to use the `bitbucket`
+delegated joining method to eliminate the need for long-lived secrets.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A Bitbucket repository you can push to.
+
+## Step 1/4. Determine Bitbucket configuration
+
+Bitbucket joining requires a number of configuration parameters that can be
+found in your repository settings. From the Bitbucket repository, navigate to
+"Repository settings", then in the sidebar under "Pipelines" select "OpenID
+Connect".
+
+From this page, note the following values:
+- Identity provider URL (<Var name="identity-provider-url" />)
+- Audience (<Var name="audience" />)
+- Workspace UUID (<Var name="workspace-uuid" />)
+- Repository UUID (<Var name="repository-uuid" />)
```

Comment:
I'm hoping to have a good variable UX here but the inline values shown in some other docs pages don't make a lot of sense once a value is entered since the variable name gets hidden. I'm not sure if this is a reasonable compromise - any thoughts from the docs team?